doc/templates/command/template.cpp

```diff
@@ -0,0 +1,42 @@
+#include "template.hpp"
+
+#include <iostream>
+
+namespace dnf5 {
+
+using namespace libdnf::cli;
+
+void TemplateCommand::set_argument_parser() {
+    auto & ctx = get_context();
+    auto & parser = ctx.get_argument_parser();
+
+    auto & cmd = *get_argument_parser_command();
+    cmd.set_short_description("A command that prints its name and arguments' name");
+
+    foo_option = dynamic_cast<libdnf::OptionBool *>(
+        parser.add_init_value(std::unique_ptr<libdnf::OptionBool>(new libdnf::OptionBool(false))));
+
+    auto foo = parser.add_new_named_arg("foo");
+    foo->set_long_name("foo");
+    foo->set_short_description("print foo");
+    foo->set_const_value("true");
+    foo->link_value(foo_option);
+
+    bar_option = std::make_unique<BarOption>(*this);
+
+    cmd.register_named_arg(foo);
+}
+
+void TemplateCommand::run() {
+    // auto & ctx = get_context();
+    std::cout << "Template Command" << std::endl;
+
+    if (foo_option->get_value()) {
```

Comment:
What about to add a comment - that will tell that the value of the options is accessible like this